master/internal/command/events.go

```diff
@@ -150,6 +245,57 @@ func validEvent(e event, greaterThanSeq, lessThanSeq *int) bool {
 	return true
 }
 
+func eventToLogEntry(ev *event) *logger.Entry {
+	description := ev.Snapshot.Config.Description
+	var message string
+	switch {
+	case ev.ScheduledEvent != nil:
+		message = fmt.Sprintf("Scheduling %s (id: %s)", ev.ParentID, description)
+	case ev.ContainerStartedEvent != nil:
+		message = fmt.Sprintf("Container of %s has started", description)
+	case ev.TerminateRequestEvent != nil:
+		message = fmt.Sprintf("%s was requested to terminate", description)
+	case ev.ExitedEvent != nil:
+		message = fmt.Sprintf("%s was terminated: %s", description, *ev.ExitedEvent)
+	case ev.LogEvent != nil:
+		message = fmt.Sprintf(*ev.LogEvent)
+	default:
+		// We rely on log entry IDs to provide pagination and since some of these events aren't actually
+		// log events we'd need to notify of them about these non existing logs either by adding a new
+		// attribute to our response or a sentient log entry or we could keep it simple and normalize
+		// command events as log struct by setting a special message.
+		// return nil, errors.New(fmt.Sprintf("event %v has no supported log message", ev))
```

Comment:
Question: if the pagination is handled in the same place as where the logs are buffered up (that is, in the eventManager), then can't we get rid of this detail?

Reply:
hmm, the goal is to clarify why we basically send, at this point, empty log messages back so that the client can also keep track